playground/api/v1/api.proto

```diff
@@ -151,6 +151,7 @@ message PrecompiledObject{
   string description = 3;
   PrecompiledObjectType type = 4;
   string pipeline_options = 5;
+  string link = 6;
```

Comment:
Can you document what the link represents?

Reply:
I added a comment for this field. Is it enough?